### PR TITLE
Silicons converted to the Ratvarian cause get an action button to communicate

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
add: Clock Cult silicons get an action button to communicate with other servants.
/:cl:

@GunHog

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 1]
```diff
@@ -107,11 +107,15 @@ This file's folder contains:
...
-			R << "<span class='warning'><b>You have been desynced from your master AI. In addition, your onboard camera is no longer active and your safeties have been disabled.</b></span>"
+			R << "<span class='boldwarning'>You have been desynced from your master AI. In addition, your onboard camera is no longer active and your safeties have been disabled.</span>"
 		S.laws = new/datum/ai_laws/ratvar
 		S.laws.associate(S)
 		S.update_icons()
 		S.show_laws()
+		var/datum/action/innate/hierophant/H = new()
+		H.Grant(S)
+		H.title = null //so it's just the borg's name
+		S << "<span class='heavy_brass'>You can communicate with other servants by using the Hierophant Network action button in the upper left.</span>"
```
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: I accidentally misspelled 'span' as 'psna' except it actually worked, ie, properly applied the span class, for some reason.

**Comment by Anonmare** [comment]
I was under the impression converted silicons not being able to communicate on Servant chat was a feature.

**Comment by ChangelingRain** [comment]
It was, but now it isn't.

File: code/game/gamemodes/clock_cult/clock_unsorted.dm [hunk 2]
```diff
@@ -20,6 +20,27 @@
...
+		return 0
+	return ..()
+
+/datum/action/innate/hierophant/Activate()
+	var/input = stripped_input(usr, "Please enter a message to send to other servants.", "Hierophant Network", "")
+	if(!input || !IsAvailable())
+		return
+
+	titled_hierophant_message(owner, input, "heavy_brass", "brass", title)
+	return
```
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_unsorted.dm]: trailing return, remove it please.

**Pull request merged**